boilerplate/lyft/golang_test_targets/download_tooling.sh

```diff
@@ -16,8 +16,8 @@ set -e
 # List of tools to go get
 # In the format of "<cli>:<package>" or ":<package>" if no cli
 tools=(
-  "github.com/vektra/mockery/cmd/mockery"
-  "github.com/lyft/flytestdlib/cli/pflags"
+  "github.com/vektra/mockery/cmd"
+  "github.com/flyteorg/flytestdlib/cli/pflags"
```

Comment:
So the go.mod and go.sum files in this folder refer to an "older" version of pflags... a 3.11 version which is still called github.com/lyft/* 

I think the easiest would be to omit changes to the golang_test_targets folder completely... let the rest of your PR get merged and a new release is created, then go back and update just that folder...

Reply:
Agreed.  I was trying to figure out how to do this all in one PR and that seemed a bit overkill to force that issue.